compiler/compile/VirtualGuard.cpp

```diff
@@ -244,20 +244,22 @@ TR_VirtualGuard::createBreakpointGuardNode
    aconstNode->setByteCodeIndex(0);
    TR::Node * flagBit = NULL;
    TR::Node *guard = NULL;
+   TR::Node * zero = NULL;
    if (TR::Compiler->target.is64Bit())
       {
       flagBit = TR::Node::create(callNode, TR::lconst, 0, 0);
       flagBit->setLongInt(comp->fej9()->offsetOfMethodIsBreakpointedBit());
-
+      zero = TR::Node::create(callNode, TR::lconst, 0, 0);
       }
    else
       {
       flagBit = TR::Node::create(callNode, TR::iconst, 0, comp->fej9()->offsetOfMethodIsBreakpointedBit());
+      zero = TR::Node::create(callNode, TR::iconst, 0, 0);
       }
 
-   guard =  TR::Node::createif(is64Bit? TR::iflcmpeq: TR::ificmpeq,
+   guard =  TR::Node::createif(is64Bit? TR::iflcmpne: TR::ificmpne,
```

Comment:
Can't both of these opcodes be assigned in the if else above using a local to remove the ternary operator?